wpa_supplicant/wpa_cli_cmds.c

```diff
@@ -3646,6 +3647,373 @@ static const struct wpa_cli_cmd wpa_cli_commands[] = {
 #endif  /* !CONFIG_ZEPHYR || (CONFIG_ZEPHYR && CONFIG_WPA_CLI)*/
 	{ NULL, NULL, NULL, cli_cmd_flag_none, NULL }
 };
+#else
+/* Compile commands without help string to spare some bytes */
+static const struct wpa_cli_cmd wpa_cli_commands[] = {
+	{"status", wpa_cli_cmd_status, NULL, cli_cmd_flag_none, NULL},
```

Comment:
Not sure what will be easier to maintain in future.. but alternatively you can do:

```
#ifdef CONFIG_WPA_CLI
#define HELP_MSG(msg) (msg)
#else
#define HELP_MSG(msg) NULL
#endif
```
And later:
```
{ "status", wpa_cli_cmd_status, NULL,
	  cli_cmd_flag_none,
	  HELP_MSG("[verbose] = get current WPA/EAPOL/EAP status") },
```

Reply:
Yeah, using a macro to compile out RO strings is good. And this commands is `nrf noup` anyway, as I didn't want to duplicate the commands, so, moved to a new file `wpa_cli_cmds.c` which doesn't exist in the opensource, so, this is fine.